### DISCUSSION TITLE
check for unsigned in postgres

### DISCUSSION THREAD

**Discussion by 3xau1o** [prisma/prisma#14344] [Q&A] [answered]
i have the following model for postgres

```prisma
model PlanDetails {
  id        String     @id @default(uuid()) @db.Uuid
  user      User?
  credits   Int        @default(1)
  status    PlanStatus
  createdAt DateTime   @default(now())
  updatedAt DateTime   @updatedAt

  @@map(name: "planDetails")
}
```

i want to add the following sql check constraint to ensure credits is a natural/positive number, is there a way to do that?

```sql
ALTER TABLE "planDetails" (
    ADD CONSTRAINT credit_lower_limit CHECK (credits >= 0)
)

```

**Comment by austincrim** [comment, marked as answer]
Hey @mindlid 👋,

You can [customize a generated migration](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/customizing-migrations) before applying it. This way, the CHECK constrain will persist in your `migrations` folder and will be applied each time you run your migrations from scratch.


You can also run one-off scripts with [`prisma db execute`](https://www.prisma.io/docs/reference/api-reference/command-reference#db-execute), but it will not be a part of your migration history.

Let me know if you hit any issues!

**Reply by 3xau1o** [reply]
Thanks, i ended customizing manually

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
